dev-server/mock/fagsak-1.json

```diff
@@ -9,6 +9,8 @@
                 "behandlingId": 1,
                 "type": "FØRSTEGANGSBEHANDLING",
                 "status": "OPPRETTET",
+                "kategori": "NATIONAL",
+                "underkategori": "EØS",
```

Comment:
Shouldn't underkategori be ORDINÆR or UTVIDET?

Reply:
ofc, my bad 🤕